03_launch_mgmt_cluster.sh

```diff
@@ -130,8 +130,8 @@ EOF
       cp "${IRONIC_AUTH_DIR}ironic-password" /opt/metal3/auth/ironic/password
       sudo mkdir -p /opt/metal3/auth/ironic-inspector
       sudo chown "$USER":"$USER" /opt/metal3/auth/ironic-inspector
-      cp "${IRONIC_AUTH_DIR}ironic-username" /opt/metal3/auth/ironic-inspector/username
-      cp "${IRONIC_AUTH_DIR}ironic-password" /opt/metal3/auth/ironic-inspector/password
```

Comment:
In case where user does not give username and password, how the pw and username is copied if these lines are removed?

Reply:
Yes. For example, if `IRONIC_INSPECTOR_USERNAME` is not provided by the user, it will [default](https://github.com/metal3-io/metal3-dev-env/blob/40e3f79b78d4f317e117b98731230a7c929f5307/lib/ironic_basic_auth.sh#L25) to IRONIC_USERNAME, which in turn generates ironic-username [here](https://github.com/metal3-io/metal3-dev-env/blob/40e3f79b78d4f317e117b98731230a7c929f5307/lib/ironic_basic_auth.sh#L12).